dear_bindings.py

```diff
@@ -246,6 +245,22 @@ def convert_header(
         'ImRect',
         'ImGuiListClipperRange'
     ])
+
+    # Mark certain types as needing placement new constructors that initialize the memory block passed into them
+    mod_mark_placement_constructor_structs.apply(dom_root, [
```

Comment:
this list was given by @ocornut . lets not depend on that.
how can i figuire out this list and/or knows when to update it?

Reply:
the ask is to provide additional detail in a comment.

Reply:
I don't list we can automatically figure out what goes in the list.
I can try to notify this project if new types appear that are likely to require constructors, otherwise people will eventually request them.